### PR TITLE
Add overview page with signup and waitlist options

### PR DESCRIPTION
**Pull request by rohitkrai03**
## Fixes 

- https://issues.redhat.com/browse/HAC-2695

## Description

- Added new overview page with signup and waitlist options.
- Cleaned up old deprecated `console` extensions and organised them.

## Type of change

- [x] Feature

## Screen shots / Gifs for design review 

https://user-images.githubusercontent.com/6041994/216279659-b712d900-545b-443f-b8f8-5a7ae715c030.mov

## Browser conformance: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by maryshak1996** [comment]
This looks great to me from the visual + UX design sides!

**Comment by rohitkrai03** [comment]
> This looks great to me from the visual + UX design sides! Only thing to check is just to make sure that the text styling for the 'Get started with CI/CD' header uses what the other overview pages uses (like App + Data Services overview for example: https://console.redhat.com/application-services/overview)

@maryshak1996 Updated the title to have the same heading level as other overview page.

Screenshot 2023-02-06 at 9 04 27 PM

**Review by rottencandy — COMMENTED** [review]
Not sure if it's important, but the last 2 icons aren't scaling well for small window sizes

File: src/components/Overview/AboutSection.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+              </Split>
+            </CardBody>
+          </Card>
+        </StackItem>
+        <StackItem>
+          <Card isLarge>
+            <CardTitle>Contact us</CardTitle>
+            <CardBody>
+              Join the{' '}
+              <a href="https://rhdevnation.slack.com/join/" target="_blank" rel="noreferrer">
```
Comment (karthikjeeyar) [inline src/components/Overview/AboutSection.tsx:128]: This link is redirecting to 404 page, maybe we should just redirect to `https://rhdevnation.slack.com`

Comment (rottencandy) [reply src/components/Overview/AboutSection.tsx:128]: We have this link to create invites https://dn.dev/DevNationSlack

File: src/components/Overview/AboutSection.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+              and deployed into a cloud hosted development environment with OpenShift.
+            </CardBody>
+          </Card>
+          <Card isPlain isCompact>
+            <CardTitle>Shift Left for DevSecOps</CardTitle>
+            <CardBody>
+              Out-of-the-box support for{' '}
+              <a href="https://slsa.dev/spec/v0.1/levels" target="_blank" rel="noreferrer">
+                SLSA Level 3
+              </a>{' '}
```
[the remark this answers is not in the document]
Comment (rohitkrai03) [reply src/components/Overview/AboutSection.tsx]: Updated to use ExternalLink.

File: src/components/Overview/AboutSection.tsx (hunk 1 above)
Comment (rohitkrai03) [reply src/components/Overview/AboutSection.tsx]: Updated the link that takes user to join page.

**Comment by rohitkrai03** [comment]
> Not sure if it's important, but the last 2 icons aren't scaling well for small window sizes 

This is happening due to extra text content on the right. Added min width and height properties for it.

**Pull request merged**